src/main/java/com/actiontech/dble/meta/DDLChildListener.java

```diff
@@ -89,6 +89,16 @@ private void updateMeta(ChildData childData) {
             return;
         }
         ClusterDelayProvider.delayBeforeUpdateMeta();
+        // just release local lock
+        if (ddlInfo.getStatus() == DDLStatus.FAILED) {
```

Comment:
When the node of DDL started execute DDL failed, other node just update the latest table structure. No need to return FAILED flag